transport/conn.go

```diff
@@ -417,6 +422,14 @@ func (c *Conn) QueryAsync(s Statement, pagingState frame.Bytes, fn func(QueryRes
 	}()
 }
 
+func (c *Conn) newStatementFromCQL(cql string) (QueryResult, error) {
```

Comment:
Nah, this should make a Statement and it's not a Conn method.

Making a shortcut like this cheats on code reusability - that is future you!